lib/components/base/Button.vue

```diff
@@ -58,21 +53,12 @@ const accentedButton = computed(() =>
     :class="{
       'icon-only': iconOnly,
       'btn-large': large,
-      'btn-danger': color === 'danger',
-      'btn-primary': color === 'primary',
-      'btn-secondary': color === 'secondary',
-      'btn-highlight': color === 'highlight',
-      'btn-red': color === 'red',
-      'btn-orange': color === 'orange',
-      'btn-green': color === 'green',
-      'btn-blue': color === 'blue',
-      'btn-purple': color === 'purple',
-      'btn-gray': color === 'gray',
       'btn-transparent': transparent,
       'btn-hover-filled': hoverFilled,
       'btn-hover-filled-only': hoverFilledOnly,
       'btn-outline': outline,
       'color-accent-contrast': accentedButton,
+      ['btn-' + props.color]: true,
```

Comment:
```
      [`btn-${color}`]: true,
```

Use string literal. Also I believe direct prop access would work here in the template.

Reply:
done